app/views/players/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for [team, player] do |f| %>
+<%= form_for [player] do |f| %>
```

Comment:
When passing `form_for` method a single object, there's no need to wrap it an Array.